DeepSpeech.py

```diff
@@ -960,6 +1027,54 @@ def main(_):
         tfv1.reset_default_graph()
         do_single_file_inference(FLAGS.one_shot_infer)
 
+    if FLAGS.finetune_lm_csv_files:
+        original_report_count = FLAGS.report_count
+        FLAGS.report_count = 0
+        original_alpha, original_beta, best_alpha, best_beta = finetune_lm_params()
+        print('[FINETUNE LM PARAMETERS] best alpha = {}, best beta = {}'.format(
+            best_alpha, best_beta
+        ))
+        if FLAGS.test_files:
+            FLAGS.lm_alpha = best_alpha
+            FLAGS.lm_beta = best_beta
```

Comment:
Instead of overwriting the global flag, refactor the called functions to be able to take these values as parameters and pass them in.